gradle/libs.versions.toml

```diff
@@ -90,7 +90,6 @@ flycoTabLayout = "io.github.h07000223:flycoTabLayout:3.0.0"
 glide = { module = "com.github.bumptech.glide:glide", version.ref = "glide" }
 glide-okhttp3 = { module = "com.github.bumptech.glide:okhttp3-integration", version.ref = "glide" }
 glide-transformations = "jp.wasabeef:glide-transformations:4.3.0"
-photoView = "com.github.chrisbanes:PhotoView:2.3.0"
```

Comment:
`PhotoView` latest version only be published to jitpack.

https://github.com/Baseflow/PhotoView#dependency
https://search.maven.org/artifact/com.github.chrisbanes.photoview/library